t/lib/NoFork.pm

```diff
@@ -4,7 +4,21 @@ BEGIN {
     *CORE::GLOBAL::fork = sub { die "you should not fork" };
 }
 use Config;
-tied(%Config)->{d_fork} = 0;    # blatant lie
+if ($Config::Config{d_fork}) {
```

Comment:
Some comments would be helpful here, to explain what is going on. What is the fix, and why is this code needed?